src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/IWorkflowRoutesProvider.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Workflows.Http.Services
```

Comment:
This has to go in the Abstractions project.

Also it doesn't provide much more than an event, or a custom startup. It could at least return the list of route entries, and the calling would add them.

Reply:
It is better to move it to the Abstract project. But if we want to  move it to Abastraction and also change the interface to return back the routes, (instead of registering them), then we have to move WorkflowRoutesEntry to the Abtraction project too. In this way, we have to move the other interfaces related to it like IWorkflowInstanceRouteEntries and IWorkflowTypeRouteEntries to the Abstraction project too.

I am afraid ahead of release, it would be too much refactoring. Should we create an issue to do it in the future, or I can go ahead with  the refactoring?